lib/index.js

```diff
@@ -1,17 +1,19 @@
 "use strict";
 
-module.exports = {
+var recommendedRulesConfig = {
```

Comment:
since this variable is only the "getify-says" recommended rules config, let's name it like this:

```js
var presetRulesConfig = {
   "getify-says": { ... }
};
```

And then in the other places we use `recommendedRulesConfig`, let's change those to `presetRulesConfig["getify-says"]`.

This will "future-proof" to if we add other presets besides my own personal opinion.